confbot_sensors/src/nodes/confbot_laser.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "confbot_sensors/nodes/confbot_laser.hpp"
+
+#include <chrono>
+#include <cmath>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <thread>
+
+#include "lifecycle_msgs/msg/transition.hpp"
+
+#include "rclcpp/rclcpp.hpp"
+
+#include "rclcpp_lifecycle/lifecycle_node.hpp"
+#include "rclcpp_lifecycle/lifecycle_publisher.hpp"
+
+#include "rcutils/logging_macros.h"
+
+#include "sensor_msgs/msg/laser_scan.hpp"
+
+#define DEG2RAD M_PI / 180.0
+
+namespace confbot_sensors
+{
+namespace nodes
+{
+
+#define DEG2RAD M_PI / 180.0
+
+ConfbotLaser::ConfbotLaser(rclcpp::NodeOptions options)
```

Comment:
```suggestion
ConfbotLaser::ConfbotLaser(const rclcpp::NodeOptions & options)
```

Reply:
Makes sense :+1:
I guess we should also update the signature in the header and the `confbot_driver` components as well.

I'll commit all these together